framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -593,195 +484,306 @@ public void standardizeVariable(AnnotatedTypeMirror type, Element elt) {
                     return;
                 }
 
-                standardizeVariable(declarationTree, type, elt);
+                atVariableDeclaration(declarationTree, type, elt);
                 return;
 
             default:
-                // It's not a variable (it might be METHOD, CONSTRUCTOR, CLASS, or INTERFACE, for
+                // It's not a local variable (it might be METHOD, CONSTRUCTOR, CLASS, or INTERFACE,
+                // for
                 // example), so there is nothing to do.
                 break;
         }
     }
 
     /**
-     * Parse and standardize the expressions in dependent types in {@code type} as if they were
-     * written at {@code localVarPath}. For example, {@code @KeyFor("field") String} is changed to
-     * {@code @KeyFor("this.field")}
+     * Viewpoint-adapt a type qualifier obtained from a contract using {@code stringToJavaExpr}.
```

Comment:
Why is "from a contract" important?  It does not seem to be used in the method body.

Reply:
The point was that this was the method to call to standardize contract annotations. I've moved this method to `Contract` class and renamed it.